modules/oe_content_extra_project/README.md

```diff
@@ -0,0 +1,10 @@
+# OpenEuropa Content Extra Project
+
+This module provides the content extra project fields.
+
+### Fields added
+  - Achievements and milestone.
+  - Gallery.
+  - Impacts.
+  - Objective.
+  - Lead contributors.
```

Comment:
What's with the indentation here?

Overall, in this readme we could mention:
- The entity type 'Project' is coming from oe_content_project.
- This module adds additional fields.
  - We could mention what these fields do, but up to you, perhaps we don't want to spend too much time.
- This module overrides form display and view display to add the new fields.